.gitignore

```diff
@@ -126,3 +126,7 @@ src/static/superhero/
 src/themes/clean
 
 src/themes/superhero
+
+# Lando
+local.env
```

Comment:
I believe this should now be `dockerfiles/lando_local.env`

Reply:
yep, oops, thanks!